cocos/ui/UITextField.h

```diff
@@ -58,6 +58,13 @@ class CC_GUI_DLL UICCTextField: public TextFieldTTF, public TextFieldDelegate
     
     virtual void onEnter() override;
     
+    /**
+     *@brief Create an empty UICCTextField.
+     
+     *@return A UICCTextField instance.
+     */
+    static UICCTextField* create();
```

Comment:
static function should be declared before all other functions. Please refer to [coding style][https://github.com/cocos2d/cocos2d-x/blob/v3/docs/CODING_STYLE.md].

Reply:
OK, I will look at code style, and change it